python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -1319,7 +1321,10 @@ def selectCoaddPsf(self, templateCoadd, warpRefList):
         # Note: ``ccds`` is a `lsst.afw.table.ExposureCatalog` with one entry per ccd and per visit
         # If there are multiple ccds, it will have that many times more elements than ``warpExpRef``
         ccds = templateCoadd.getInfo().getCoaddInputs().ccds
-        psfRefSize = templateCoadd.getPsf().computeShape().getDeterminantRadius()*sigma2fwhm
+        templatePsf = templateCoadd.getPsf()
```

Comment:
Is saving `templatePsf` really needed here or can we use it on the fly like all the other instances? You don't assign `calPsf` for instance, you just use `cal.getPsf().stuff`